ios/FluentUI/Popup Menu/PopupMenuProtocols.swift

```diff
@@ -38,7 +38,7 @@ public protocol PopupMenuTemplateItem: AnyObject {
  You can use `UITableViewCell` conforms to this protocol for customization.
```

Comment:
Should we improve this documentation now that we don't enforce that it's of type UITableViewCell to explain it needs to be?

Reply:
I think the comment already tries to say that, but tried to make it more explicit